src/components/footer.js

```diff
@@ -67,21 +67,77 @@ const CustomWrap = styled.div`
   text-align: center;
 `;
 
-const Footer = () => (
-  <FooterWrap>
-    <div className={wrap} />
-    <CustomWrap />
-    <div className={wrap}>
-      <Dropdown>
-        <div className={dropdownContent}>
-          <Element>1</Element>
-          <Element>2</Element>
-          <Element>3</Element>
+class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentLanguage: 'en',
+    };
+  }
+
+  changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
+    this.setState({ currentLanguage: lng });
+  };
+
+  chooseFlag = (lng) => {
+    if (lng === 'en') {
+      return (
+        <img
+          className="icon"
+          src="https://cdn.pg.edu.pl/ekontakt-updated-theme/images/language/en_GB.png"
```

Comment:
Potencjalnie spoko, ale nie wolałbyś wgrać swoich własnych flag żeby nie być uzależnionym od _stron_ trzecich? :D Potencjalnie zwolni to twoją apkę albo może ją nawet wysypie gdy u nich się coś zmieni,

Reply:
Faktycznie te dwie flagi można zapisać w pliki repo :)